tickets/src/components/content/EventContent.js

```diff
@@ -186,6 +199,25 @@ export const mapStateToProps = (
       description,
       location,
     }
+    // TEMPORARY: HARD CODE VALUES FOR NFT EVENT
+    if (lockAddress === '0x5Cd3FC283c42B4d5083dbA4a6bE5ac58fC0f0267') {
```

Comment:
Don't you mean `0x5865Ff2CBd045Ef1cfE19739df19E83B32b783b4`?

Reply:
Oups! thanks for the sharp eye!